### PR TITLE
telegram-desktop: update to 4.2.4

### PR DESCRIPTION
**Pull request by wrathgewp**
Testing the changes
- I tested the changes in this PR: **YES**

Local build testing
- I built this PR locally for my native architecture, aarch64

### CODE REVIEW

File: srcpkgs/telegram-desktop/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'telegram-desktop'
 pkgname=telegram-desktop
-version=4.1.1
-revision=1
+version=4.2.4
+revision=2
```
Comment (Johnnynator) [inline srcpkgs/telegram-desktop/template:4]: Revision should be reset (or kept) at 1 for every new version, it only needs to be increased when rebuilding without changing the version

**Review by Johnnynator — CHANGES_REQUESTED** [review]

File: srcpkgs/telegram-desktop/template (hunk 1 above)
Comment (classabbyamp) [inline srcpkgs/telegram-desktop/template:4]:
```suggestion
revision=1
```
revision should only be incremented if the package keeps the same version and needs to be rebuilt

Comment (wrathgewp) [reply srcpkgs/telegram-desktop/template]: Thank you for your quick answer. I fixed it, sorry for the error.

**Comment by paper42** [comment]
commit message should be just `telegram-desktop: update to 4.2.4` and shouldn't contain the body

**Comment by wrathgewp** [comment]
> commit message should be just `telegram-desktop: update to 4.2.4` and shouldn't contain the body

I fixed it now. Is this correct?

**Review by Johnnynator — APPROVED** [review]

**Pull request merged**